src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -221,11 +221,16 @@ private static String urlEncodePair(String k, String v) throws UnsupportedEncodi
 		return conn;
 	}
 
-	private static javax.net.ssl.HttpsURLConnection createPostConnection(String url, String query, String apiKey) throws IOException {
+	private static javax.net.ssl.HttpsURLConnection createPostConnection(String url, String query, String apiKey, Boolean isRating) throws IOException {
 		javax.net.ssl.HttpsURLConnection conn = createEasyPostConnection(url, apiKey);
 		conn.setDoOutput(true);
 		conn.setRequestMethod("POST");
-		conn.setRequestProperty("Content-Type", String.format("application/x-www-form-urlencoded;charset=%s", CHARSET));
+		if (!isRating) {
+			conn.setRequestProperty("Content-Type", String.format("application/x-www-form-urlencoded;charset=%s", CHARSET));
+		}
+		else{
+			conn.setRequestProperty("Content-Type", String.format("application/json"));
```

Comment:
I'm not a massive fan of this `isRating` change, but if we need to tinker with the internals like this then I guess there's not much to do about. This conditional is the primary purpose of this variable, right?